docs/troubleshooting_and_faq/collections.md

```diff
@@ -0,0 +1,57 @@
+# Collections
```

Comment:
Some of these will also apply to other tests - connect, endpoint-urls, etc. For example, all could return a 500 error for the same reason. Maybe change the title of this to something like `Issues with mp-test`? (And the build section to `Issues with mp-build` to match?)